work.html

```diff
@@ -59,7 +59,7 @@
     
     <style type="text/css">@import url("https://assets.mlcdn.com/fonts.css?version=1755584");</style>
     
-    <link rel="stylesheet" href="css/styles.min.css">
+    <link rel="stylesheet" href="css/styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="css/styles.min.css">
```

Reply:
@oromama Important note, in the future do not remove the minified css like you did here. There is a minify.sh script in tools dir that you need to run that minifies the css.